### DISCUSSION TITLE
If a collaborator takes up my last seat, Can't add the collaborator to multiple repositories.

### DISCUSSION THREAD

**Discussion by Next-Layer** [community/community#35680] [Other Feature Feedback, Questions, & Ideas]
My organisation recently contracted a development agency to do some works for us. I need to add a few of their devs as collaborators onto our repositories. 

This all works fine until I am trying to add the last collaborator. She takes up the last seat in our subscriptions. Was able to add her to the first repository. But When I try to add her to the next repository, I was asked to purchase more seats. Previous collaborators only take up one seat despite being invited into multiple repositories, so I shouldn't be required to purchase extra seat for the last collaborator.  I think this is a bug that needs solving, especially for small organisations.

I wonder if there are ways to work around this bug for the time being.

Thanks for your help in advance.

**Comment by jsoref** [comment]
Open a ticket with: https://support.github.com

**Comment by ghost** [comment]
I'm also having this issue.  Was there any resolution or were you forced to buy more seats?

**Comment by ghost** [comment]
The workaround seems to be that you currently always need one seat open.  As soon as you purchase a seat, you can add that person to more repositories without them taking up more seats.  The code is always being forced to check for an open seat even if the person added isn't going to take up another seat.

**Reply by Next-Layer** [reply]
The way I got around it, was to temporarily kick a member of the organisation out. That leaves me with the extra license to add the external contributors to the repositories I wanted to.

After I sent out the invitation, the algorithms determines that these multiple invitations (to different repositories) )only takes up one license and I then have one license spare. I then added the member back in. As the member only requires one invitation, it doesn't cause any problems.

**Comment by iRonin** [comment]
It's October 2024 and this is still happening…

**Reply by kirby-jack** [reply]
same

**Reply by mnogely-dtml** [reply]
It's August 2025 and I just realized this is a thing. Now I'm realizing it is a thing since 2022

**Comment by jsoref** [comment]
We generally don't use the collaborators feature. Instead, I've set the default permissions for our organization to be nothing. I have teams for our employees and teams for any collaborating groups. When I create a repository, I set permissions for our main teams to give them read/write access, and if we need to give access to people who aren't employees, we just add their team as well.

Since, afaict, seats cost the same, whether for a collaborator, or for an employee, I saw no reason to use the collaborator permissions.

This means we don't trip on the problem you're tripping on.

Yes, this approach means you have to be more careful about repository permissions, but it doesn't seem like a big problem. You can allow people to fork repositories and then only give them read access to the main repository, and rely on pull requests from their forks. Or you can set up CODEOWNERS and branch protection.
